libs/challenge-registry/challenge-search/src/lib/challenge-search-filters-values.ts

```diff
@@ -55,3 +55,84 @@ export const challengeStartYearRangeFilterValues: FilterValue[] = [
     active: false,
   },
 ];
+
+export const challengeStatusFilterValues: FilterValue[] = [
+  {
+    value: 'active',
+    label: 'Active',
+    active: false,
+  },
+  {
+    value: 'upcoming',
+    label: 'Upcoming',
+    active: false,
+  },
+  {
+    value: 'completed',
+    label: 'Completed',
+    active: false,
+  },
+];
+
+export const challengeDifficultyFilterValues: FilterValue[] = [
+  {
+    value: 'GoodForBeginners',
+    label: 'Good For Beginners',
+    active: false,
+  },
+  {
+    value: 'Intermediate',
+    label: 'Intermediate',
+    active: false,
+  },
+  {
+    value: 'Advanced',
+    label: 'Advanced',
+    active: false,
+  },
+];
+
+export const challengeSubmissionTypesFilterValues: FilterValue[] = [
+  {
+    value: 'DockerImage',
+    label: 'Docker Image',
+    active: false,
+  },
+  {
+    value: 'PredictionFile',
+    label: 'Prediction File',
+    active: false,
+  },
+  {
+    value: 'Other',
```

Comment:
The checkbox doesn't display a check when a user clicks on the text, despite the filter being applied:

https://user-images.githubusercontent.com/9377970/207993125-120f4c9f-2766-4340-a33d-04cb7811801d.mov


Clicking in the actual box _will_ display the checkmark.

This behavior is only seen with "Other" -- the other choices behave as expected (that is, clicking on the box or text will apply/remove the filter).

Reply:
Good catch! It is a bug. I think it's because the label names are overlapped in different filters, like Others - I will fix this. Thank you. 
![Recording 2022-12-16 at 10 14 19](https://user-images.githubusercontent.com/73901500/208129025-69bbf2cc-d777-489f-b86e-3debadc0f0d1.gif)

Reply:
Fixed
<img src="https://user-images.githubusercontent.com/73901500/208136531-664fd91a-0ec5-45bb-a09f-6aeefe13e1d3.gif" height="250">